.github/workflows/flake_finder.yml

```diff
@@ -20,8 +20,14 @@ jobs:
         include:
           # Recentness of K8s versions are limited by kindest/node image releases
           - k8s_version: 1.18.15
+          - k8s_version: 1.18.15
```

Comment:
may be it's a little bit "animal" but what do you think of adding the whole matrix of k8s versions in L18?

In such way we would also test lighthouse which could potentially have issues with different versions of the K8s API and their behaviors

Anyway, just up for discussion, in it's current form is the most urgent thing that we need to test.

Reply:
I think this makes sense, but maybe in https://github.com/submariner-io/lighthouse/pull/503?